### ISSUE TITLE
Exception: invalid literal for int() with base 10: '//mainnet.infura.io/v3/b7a001b2e2674c46bd21367fb4910b73'

### ISSUE THREAD

**Issue by Arwinjd** [cleanunicorn/karl#72]
Hi,
I recently created an account on Infura and used my respective mainnet address with karl and this is what I'm receiving back: 

`root@work-pc:~# karl --rpc https://mainnet.infura.io/v3/b7a001b2e2674c46bd21367fb4910b73`
`[CLI] Exception: invalid literal for int() with base 10: '//mainnet.infura.io/v3/b7a001b2e2674c46bd21367fb4910b73'`

Any suggestions?

Thank you,

Kalen

**Comment by cleanunicorn** [comment]
I see the problem and I will change the `--rpc` argument to parse infura links differently.

At the moment this is the help and it's not useful anymore because Infura added rate limiting and API keys.
```
  --rpc HOST:PORT / ganache / infura-{mainnet, rinkeby, kovan, ropsten}
                        Custom RPC settings (default: None)
```

**Comment by cleanunicorn** [comment]
It should be fixed in version `0.8.0`.
